eventstore/storethread.go

```diff
@@ -145,19 +145,13 @@ func (a *singleThreadAdapter) storeToThread(wg *sync.WaitGroup) {
 		case <-a.closeChan:
 			log.Infof("closing store-to-thread flow on thread %s", a.threadID)
 			return
-		case event, ok := <-l.Channel():
+		case node, ok := <-l.Channel():
 			if !ok {
 				log.Errorf("ending sending store local event to own thread since channel was closed for thread %s", a.threadID)
 				return
 			}
-			n, err := event.Node()
-			if err != nil {
-				log.Fatalf("error when generating node for own log for thread %s: %v", a.threadID, err)
-			}
 			ctx, cancel := context.WithTimeout(context.Background(), addRecordTimeout)
-			log.Debugf("adding new local store event to own log from entityid: %s", event.EntityID())
-			_, err = a.api.AddRecord(ctx, a.threadID, n)
-			if err != nil {
+			if _, err := a.api.AddRecord(ctx, a.threadID, node); err != nil {
```

Comment:
Much less work done here.
As mentioned before, now the `StoreThread` receives the `node` just ready to be added as a record. The `EventCodec` already does the work that was done here (doing some wierd `event.Node()` call that doesn't exist anymore).